packages/portal-proto/src/components/Modals/SetModals/SavedSets.tsx

```diff
@@ -78,17 +78,19 @@ const SavedSets: React.FC<SavedSetsProps> = ({
         id: "select",
         header: "Select",
         cell: ({ row }) => (
-          <Checkbox
-            size="xs"
-            classNames={{
-              input: "checked:bg-accent checked:border-accent",
-            }}
-            aria-label={`Select/deselect ${row.original.setId}`}
-            {...{
-              checked: row.getIsSelected(),
-              onChange: row.getToggleSelectedHandler(),
-            }}
-          />
+          <>
+            <Checkbox
+              size="xs"
+              classNames={{
+                input: "checked:bg-accent checked:border-accent",
+              }}
+              aria-label={`Select ${row.original.setId}`}
+              {...{
+                checked: row.getIsSelected(),
+                onChange: row.getToggleSelectedHandler(),
+              }}
+            />
+          </>
```

Comment:
does not need to be in `<></>` 

aria-label={`Select ${row.original.setId}`} should be whatever the user is selecting and no additional text, if the value is visible on the page it should be aria-labelledby instead and point to that value